velox/common/compression/tests/CMakeLists.txt

```diff
@@ -16,14 +16,5 @@ add_executable(velox_common_compression_test CompressionTest.cpp)
 add_test(velox_common_compression_test velox_common_compression_test)
 target_link_libraries(
   velox_common_compression_test
-  velox_common_compression
-  Boost::regex
-  velox_link_libs
-  Folly::folly
-  ${TEST_LINK_LIBS}
-  gflags::gflags
-  gtest
-  gtest_main
-  gmock
-  glog::glog
-  fmt::fmt)
+  PUBLIC velox_link_libs ${TEST_LINK_LIBS}
+  PRIVATE velox_common_compression velox_exception gtest gtest_main)
```

Comment:
Nit: isnt gtest part of TEST_LINK_LIBS ?

Reply:
It would be but looking at it now I am pretty sure that TEST_LINK_LIBS is empty as it is defined in dwio/... and this is configured before that. Also we should remove that var (and similar ones) and replace it with an interface target.

Reply:
(yes it's empty) 
![image](https://github.com/facebookincubator/velox/assets/16141871/aaff6838-229c-4646-bdda-549487377434)

Reply:
Lets just remove that then..